File: Ajay_thakur/Collection_framework/06_02_2025/04.java
/*
 * Program to implement previous() methods
 */

import java.util.*;

class Main{

    public static void main(String[] args) {
        
        ArrayList<String> l = new ArrayList<>();

        l.add("rashmika");
        l.add("deepika");
        l.add("katappa");
        l.add("bahubali");
        
        System.out.println(l);

        ListIterator itr = l.listIterator();

        while(itr.hasNext()){

            String i = (String)itr.next();
            System.out.println(i);
        }
        
        while(itr.hasPrevious()){

            String i = (String)itr.previous();
            System.out.println(i);
        }

        System.out.println(l);
    }
}

File: Javascript/02_05_2025/04.js
var students = [

    {id:101,name:'deepika',marks:99},
    {id:102,name:'rashmika',marks:89},
    {id:103,name:'katappa',marks:85},
];

console.log(students);

console.log(students[0]);

console.log(students[0].name);

console.log(students[2]['name']);

for(var student of students){

    console.log("id is " + student.id + " name is " + student.name);
}